src/txdb.cpp

```diff
@@ -62,7 +63,14 @@ uint256 CCoinsViewDB::GetBestBlock() const {
     return hashBestChain;
 }
 
-bool CCoinsViewDB::BatchWrite(CCoinsMap &mapCoins, const uint256 &hashBlock) {
+uint256 CCoinsViewDB::GetFinalTx() const {
+    uint256 hashFinalTx;
+    if (!db.Read(DB_LAST_TX, hashFinalTx))
+        return uint256();
```

Comment:
Style-nit: same line or `{}`ed. (Not mentioning these from here on.)

Reply:
Fixed.

Reply:
I looked through and caught a few others.